test/UnitTest.cs

```diff
@@ -3141,4 +3141,769 @@ public void TestParenthesisError()
             RawParser.ParseSqlStatements(sql);
         }
     }
+
+    [TestClass]
```

Comment:
can we add a separate file for this? unittest is already too big

Reply:
Yes, but how do I integrate into existing framework? If all of NIST stuff is moved into NistTests.cs, can they be called from the existing one by invoking RunNistTests()?

Reply:
Moved to test/NistTests.cs. The string compare tests are in their own file, StringCompare.cs.